### DISCUSSION TITLE
文献整理和分类

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#1078] [General]
> 感谢快速的回复，这个插件真的特别好，感觉再继续完善，我真的可以放弃我的obsidian联动了
> 
> 1. 个人觉得article title、Journal name、IF、authors、published year、tags这几个如果能的话都能加上，如果是编辑的时候能够点击某个按钮一键插入是最好的，如果不需要可以不插入，这样自由度会比自动插入更高
> 2. 设置长一些会好一些，至少能显示的更清楚，思维导图就是为了显示框架结构，连框架的分支都看不全就看不清结构了，思维导图的意义就失去了，或者更好的是每一个节点的字段宽度可以自由拖拽来调节，这样自由度会更高，但是不知道好不好实现，实在不好实现，就做成字段允许长度大一些吧，辛苦啦
> 3. 还有一个小建议，就是笔记是否可以设置模板然后有个按钮可以一键插入模板，比如说我在看文献的时候有一个习惯的整理逻辑和框架习惯（比如B站某up主的方式，类似于下图思维导图的结构），然后把每个文献按照这个逻辑整理，或者看了一堆文献时按照某一个逻辑框架去梳理，插件能加入一个按钮允许我插入一个模板，我每次新建笔记的时候可以选择某一个模板。其实我现在使用咱们这个插件的方式就是我建了一个模板在一个单独的notes里，每次我看一个文献记笔记的时候，我就把模板copy-paste过来，再添加，虽然也没有多麻烦，但是感觉没有在插件加入一个模板按键的方式更方便、工作流不够流畅
> 
> <img alt="image" width="1299" src="https://user-images.githubusercontent.com/105227871/167526321-40534dc3-d69d-4e11-865e-8057266e502e.png">
> 
> 再次感叹插件很棒，再次给您点赞

请问这个的笔记模版在哪里下载？看这个思维导图框架也挺好看。

另外说几句自己的心得：这两天我也是在思考Better Note如何用来整理某一类（主题）的文献，毕竟我一开始是抱着这个目的找到的这款笔记软件。学习完之后发现BN更多是单篇文献记笔记，本来以为是要在主笔记中采用双链、引用单篇文献的笔记，不知道如何搭建主笔记的逻辑框架，搜索后来到了该帖。
看了这个帖子的回复后，我知道了可能要另想办法，做文献的整理和分类工作。

_Originally posted by @iampearl918 in https://github.com/windingwind/zotero-better-notes/issues/1#issuecomment-2278957081_

**Comment by windingwind** [comment]
@iampearl918 

勿回复很久以前的issue。有相关内容请单独开discussion讨论，谢谢

关于上述内容的回复：

> 学习完之后发现BN更多是单篇文献记笔记，本来以为是要在主笔记中采用双链、引用单篇文献的笔记，不知道如何搭建主笔记的逻辑框架，搜索后来到了该帖。

bn的功能分为多个不同的模块，其中笔记模板主要是从一个/多个文献收集归纳信息到当前笔记，在许多帖子中已经充分讨论，不再赘述；双链功能即你所提到的用法，很多网上的教程、经验分享不涉及这部分，可能是因为大多数人并不习惯真正双链的笔记结构。bn设计的初衷即是每个文献产生一个或多个小笔记，互相双链关联并最终根据当前主题归总到一个笔记中。

**Reply by iampearl918** [reply]
感谢解惑！摸索使用了一段时间，再来交流请教一下：


> 笔记模板主要是从一个/多个文献收集归纳信息到当前笔记，

感谢！这个突然给了我启发，我可以建一个笔记，将多个不同文献中自己需要的东西归纳到一起。虽然不是采用双链Note的格式，而是将不同文章中的内容直接放到一个Note，但似乎达到了我想要的效果。


> 双链功能即你所提到的用法，很多网上的教程、经验分享不涉及这部分，可能是因为大多数人并不习惯真正双链的笔记结构。bn设计的初衷即是每个文献产生一个或多个小笔记，互相双链关联并最终根据当前主题归总到一个笔记中。

这个是否就是您在betternote说明书中讲的细粒度、中粒度、粗粒度文献笔记方法？
